features/admin.users.v1/components/user-profile.tsx

```diff
@@ -2636,13 +2661,140 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
+        accountState === AccountState.PENDING_AP;
+
+    /**
+     * Initiates a recovery process based on the account's locked reason or account state.
+     *
+     **/
+    const handleResendCode = async () => {
+        setIsSubmitting(true);
+
+        try {
+            let recoveryScenario: string = null;
+
+            // If account is locked determine the recovery scenario based on the locked reason.
+            if (accountLocked && accountLockedReason) {
+                recoveryScenario = ACCOUNT_LOCK_REASON_TO_RECOVERY_SCENARIO_MAP[accountLockedReason];
+            }
+
+            // If account is not locked determine the recovery scenario based on the account state.
+            if (!accountLocked && isPendingAskPassword) {
+                recoveryScenario = ACCOUNT_STATE_TO_RECOVERY_SCENARIO_MAP[accountState];
+            }
+
+            if (recoveryScenario) {
+                const requestData: ResendCodeRequest = {
+                    properties: [
+                        {
+                            key: "RecoveryScenario",
+                            value: recoveryScenario
+                        }
+                    ],
+                    user: {
+                        realm: user[ SCIMConfigs.scim.systemSchema ]?.userSource,
+                        username: user?.userName
+                    }
+                };
+
+                await resendCode(requestData);
+                onAlertFired({
+                    description: t("user:profile.notifications.resendCode.success.description"),
+                    level: AlertLevels.SUCCESS,
+                    message: t("user:profile.notifications.resendCode.success.message")
+                });
+            } else {
+                onAlertFired({
+                    description: t("user:profile.notifications.resendCode.inValidRecoveryScenarioError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("user:profile.notifications.resendCode.inValidRecoveryScenarioError.message")
+                });
+            }
+        } catch (error: any) {
+            onAlertFired({
+                description: error.response?.data?.description ||
+                    t("user:profile.notifications.resendCode.genericError.description"),
+                level: AlertLevels.ERROR,
+                message: t("user:profile.notifications.resendCode.genericError.message")
+            });
+        } finally {
+            setIsSubmitting(false);
+        }
+    };
+
     return (
         !isReadOnlyUserStoresLoading && !isEmpty(profileInfo)
             ? (<>
                 {
                     (accountLocked || accountDisabled) && (
                         <Alert severity="warning">
                             { t(resolveUserAccountLockedReason()) }
+                            { showResendLink && (
+                                <>
+                                    &nbsp;
```

Comment:
get the space through css file rather thanusing `&nbsp`

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/399e7573e645d5c76ea0e9c6b3a47780a2c40019 .